.github/workflows/ci.yml

```diff
@@ -11,9 +11,9 @@ jobs:
   build:
     strategy:
       matrix:
-        temurin_tag:
-          - 8u322-b06-jdk
-          - 11.0.14.1_1-jdk
+        remoting_tag:
+          - 4.13.2-1-jdk8
```

Comment:
will we need to update these values as [new versions are released](https://github.com/jenkinsci/docker-inbound-agent/releases)?

Reply:
Yes, but I wonder if something like Dependabot could do that for us?